### ISSUE TITLE
use keyboard tab to trigger panel open, the next keyboard focus cannot land on the first item in GroupedList as their body

### ISSUE THREAD

**Issue by chihch** [microsoft/fluentui#12933]
### Environment Information

- **Package version(s)**: 7.94.0
- **Browser and OS versions**: windows 10

### Describe the issue:

Use keyboard tabbing to trigger a panel to open, and notice that the next focus item is not on the first item of GroupedList in the panel body. The focus jumps to the element in the footer of panel.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/chihch/pen/wvKqyvN

#### Actual behavior:

if using a GroupedList as panel body, the default focus cannot land on the first item in the groupedlist. It jump to the footer area to find focusable item and focus to it. In the codepen, I put together a regular button as panel body vs. a groupedList as panel body. You can turn on and off to see the issue. Keep in mind that this is using keyboard navigation.

#### Expected behavior:

if GroupedList is used in panel body, the first focused item after panel is opened by keyboard tabbing is the first focusable item in the footer section

**Comment by joschect** [comment]
@chihch Thanks for the submission! I can see what you're talking about in your codepen. This is happening because the focusable items in the header are in a focusZone. There should be some way to put focus on the focus zone but that doesn't seem to be the case. I tried several workarounds but was unable to find one. Some of this could be caused by the GroupedList doing delayed rendering, but as GroupedList itself doesn't have a focusable method there's no way to force focus into it easily after the fact. We'll get this added to our backlog.

**Comment by chihch** [comment]
got it. thanks for your fast explanation and investigation. Really appreciate your help.

**Comment by chihch** [comment]
can we reopen this item? This is an accessibility issue. thanks so much.
